src/datatypes/interval.coffee

```diff
@@ -48,8 +49,11 @@ module.exports.Interval = class Interval
     )
 
   includedIn: (other) ->
-    if not (other instanceof Interval) then throw new Error("Argument to includedIn must be an interval")
-    other.includes @
+    # For the point overload, this operator is a synonym for the in operator
+    if not (other instanceof Interval)
+      @.contains other
```

Comment:
This doesn't look like it gets hit. Point overload gets converted into `In` in the ELM. The `In` operator in `overloaded` calls `doContains` which calls the interval `contains` function.

Reply:
Disregard these comments. After running `cake build-all` the functionality looks correct.

Reply:
This only gets hit in tests that explicitly call `IncludedIn`